src/services/pythonServer.ts

```diff
@@ -22,7 +22,7 @@ class PythonServer {
     return this.server
   }
 
-  resolve(promise: Promise<any>, setter: any) {
+  resolve(promise: Promise<any>, setter?: any) {
```

Comment:
Is the `setter` really optional? Seems like you always execute it on `data.success`

Reply:
The idea was that sometimes it is not required to get any feedback from the server as the server side commands don't return any useful data. However, later on I decided it was better to actually implement some return statuses. 
This won't be optional on a future PR